dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -20,19 +20,23 @@ def index
       application_data = empty_application_data
 
       ROLES.each do |role|
-        apps = get_applications_by_role(role, include_associations: false).
-          select(:status, "count(id) AS total").
-          group(:status)
+        apps = get_applications_by_role(role, include_associations: false)
+
+        apps_statuses = apps.select(:status, "count(id) AS total").group(:status)
```

Comment:
What is `apps_statuses` used for? Should it also be used on line 37?

Reply:
oo I split up the old line 23 because after grouping by status, I wasn't able to find the enrollment. So I wanted the list of applications for enrollments but didn't want to change too much other logic here. I'm definitely open to naming suggestions!

Reply:
Gotcha! Thank you, totally looks good to me, just wanted to clarify!

Reply:
No problem! This tripped me up for a while 😅